service/controller/v12/cloudconfig/spec.go

```diff
@@ -4,6 +4,12 @@ import (
 	"github.com/aws/aws-sdk-go/service/kms"
 )
 
+const (
+	// CloudConfigVersion defines the version of k8scloudconfig in use.
+	// It is used in the main stack output and S3 object paths.
+	CloudConfigVersion = "v_3_3_2"
```

Comment:
This is the new constant.

Reply:
May be an idea to put it in k8scloudconfig repo.

Reply:
Good idea but going with this. It improves the situation as there is now only a single constant and no changes to tests.

We need to make more k8scc changes for managed services so we can come back to this.